eng/templates/README.md

```diff
@@ -7,7 +7,7 @@
 ### To install the template (one-time step), run following commands:
 
 ```
-dotnet new -i `Path to SDK repo`\eng\templates\Azure.ResourceManager.Template
+dotnet new install `Path to SDK repo`\eng\templates\Azure.ResourceManager.Template
```

Comment:
I changed this because when I run this command, I was told this `-i` or `--install` flag is deprecated, please use `dotnet new install` instead